src/framework/theme/components/select/select.spec.ts

```diff
@@ -281,6 +281,11 @@ describe('Component: NbSelectComponent', () => {
     expect(selectButton.textContent).toEqual(selectedOption.value.toString());
   }));
 
+  it(`should not call dispose on initialized resources`, () => {
```

Comment:
on _un_ initialized?

Reply:
Right, fixed in d8a2d4e409feddf1798a97ffec5895d6a1be8fe3.